cmd/juju/charmhub/download.go

```diff
@@ -332,7 +332,7 @@ func (c *downloadCommand) suggested(ser string, channel string, releases []trans
 	series := set.NewStrings()
 	for _, rel := range releases {
 		if rel.Channel == channel {
-			series.Add(rel.Platform.Series)
+			series.Add(rel.Base.Channel)
```

Comment:
I think here we would need to convert the channel that we receive back to a series.

Reply:
Something else is going on, downloadSuite.TestRunWithUnsupportedSeries is checking for series to be listed and passes.  Therefore something else is going on, as your comment should be true based on the line.  Will investigate.

Reply:
done